frontend/src/metabase/core/components/AccordionList/AccordionList.jsx

```diff
@@ -181,6 +185,12 @@ export default class AccordionList extends Component {
     }
   }
 
+  _getRootContainer() {
```

Comment:
do we need this function at all? Can we, in theory, pass the same ref to both elements whether they're virtualized list or not.

@oisincoveney did this when refactoring ref not too long ago.

You can probably pass `listRootRef` to both root containers. By using a function you can set the element to multiple refs there.

https://github.com/metabase/metabase/blob/f4347d984c89aee9a12bc5445907bfef52683b42/frontend/src/metabase/components/ExplicitSize/ExplicitSize.tsx#L249-L259

Reply:
That might work, I'll give that a try once I get tests to pass with the current version 👍🏻

Reply:
@WiNloSt good news: using the same `listRootRef` for both container works.

bad news: `elementRef` does not exist in the NPM package of `react-virtualized` (even though it exists in the source code on the GitHub tag!), I confirmed that `package.json` version resolves to `9.22.5`. I updated the PR to use the `_scrollingContainer` instead. I just realized that the focus() doesn't work thanks to a Loki test failure.

Reply:
I'm not sure I follow. That `elementRef` is just a variable we defined. It could be any other name.
https://github.com/metabase/metabase/blob/f4347d984c89aee9a12bc5445907bfef52683b42/frontend/src/metabase/components/ExplicitSize/ExplicitSize.tsx#L79

The point I wanted to show you is how we could leverage function ref, rather than passing a ref which only works with one single ref.